File: 01. Lenguajes de proposito general/02. Programación básica/08. Estructuras/01. Condicionales/04. Switch/Java/Switch.java

package prueba;

public class Prueba {
    
    public static void main(String[] args) {
       
        // Estandar
        switch (expresión) {
            case valor1:
                // Contenido 1
                break;
            case valor2:
                // Contenido 2
                break;
            case valorN:
                // Contenido N
                break;
            default:
                // Contenido default
        }

        // Con retorno
        tipo retorno = expresión == valor1 ? expresión1 : expresión == valor2 ? expresión2 : expresión == valorN ? expresiónN : expresiónDefault;
       
    }
    
}

File: 01. Lenguajes de proposito general/01. Programación básica/09. Estructuras/01. Condicionales/04. Switch/MathLab/Switch.m

% Estandar
switch expresión
    case valor1
        %  Contenido 1
    case valor2
        %  Contenido 2
    case valorN
        %  Contenido N
    otherwise
    	%  Contenido default
end

%  Con retorno
retorno = NaN
switch expresión
    case valor1
        retorno = expresión1
    case valor2
        retorno = expresión2
    case valorN
        retorno = expresiónN
    otherwise
    	retorno = expresiónDefault
end